ech/ech_cloudflare_test.c

```diff
@@ -0,0 +1,180 @@
+/* ech_cloudflare_test.c
```

Comment:
I would prefer this in the `tls` directory named `client_ech.c`. The certificates should go in `/certs` with names having `ech` in them.